docs/Data/Lists/Init.md

```diff
@@ -0,0 +1,28 @@
+# `Init(list)`
+
+Extracts the elements of a non-null, non-empty list excluding the last element.
+
+## Arguments
+
+* `list (List<T>)`: The list.
+
+## Returns
+
+* `(List<T>)`: Elements in the list excluding the last.
+
+## Throws
+
+* `ArgumentException` if the `list` is not a non-empty `List`.
+
+## Example
+
+```csharp
+Init(new List<int>());
+// => ArgumentException("list must be a not empty List")
+
+Init(new List<int> { 0 });
+// => List<int> { 0 }
```

Comment:
Is this result correct? In a single-element list, isn't the first element also the last element, which line `3` says is excluded?

Reply:
That is correct.

Reply:
Lists in functional scope are written as `[head, tail]`. If `tail` in this case does not exist, then `init` and `head` are effectively the same as lists are read head to tail and not tail to head.